extension/src/test/suite/setup/index.test.ts

```diff
@@ -577,7 +577,7 @@ suite('Setup Test Suite', () => {
       mockExecuteCommand.restore()
       mockRunSetup.restore()
       stub(config, 'isPythonExtensionUsed').returns(false)
-      stub(config, 'getPythonBinPath').resolves(join('python'))
+      stub(config, 'getPythonBinPath').returns(join('python'))
```

Comment:
[F] This was the cause of [these test warnings](https://github.com/iterative/vscode-dvc/actions/runs/4856306390/jobs/8655732780#step:7:677).